README.md

```diff
@@ -54,28 +56,41 @@ def f_0():
 
 f_0()
 ```
-is a `python` program generated with [`string_to_code.to_python3`](./string_to_code/to_python3.py) [displaying](https://www.online-python.com/jgzNiCAvxR) `Hello, World!`.
 
+is a `python` program generated with
+[`string_to_code.to_python3`](./string_to_code/to_python3.py)
+[displaying](https://www.online-python.com/jgzNiCAvxR) `Hello, World!`.
 
 In order to generate a code in `your_favourite_language` just call the function `string_to_code.to_your_favourite_language.proc` with the string which you want to display. [`examples`](./examples) show some basic usage.
 
-# Adding a new language
+## Adding a new language
+
+If you think that [`string_to_code`](./string_to_code) is missing some
+language, feel free to add it - it is simple.
+If you want to add support of let's say `turbo_snake`,
+you will need to add three files:
+
+  - `to_turbo_snake.py` into [`string_to_code`](./string_to_code).
+    This module should contain a function `proc` taking two arguments:
+    - `in_str` being the string which the generated code should display,
+    - `gen_function_names=None` being a generator which yields the function names in generated code.
+
+    Most likely you will apply [`core.py`](./string_to_code/core.py) module.
+    Have a look at the existing `proc` functions.
 
-If you think that [`string_to_code`](./string_to_code) is missing some language, feel free to add it - it is simple.
-If you want to add support of let's say `turbo_snake`, you will need to add three files:
-- `to_turbo_snake.py` into [`string_to_code`](./string_to_code). This module should contain a function `proc` taking two arguments:
-  - `in_str` being the string which the generated code should display,
-  - `gen_function_names=None` being a generator which yields the function names in generated code.
+  - `setup_turbo_snake.py` into [`tests`](./tests). This file is used for tests. It should contain a function `get_test_data()` returning a `Language` object having four fields:
+    - `tool_names`: a list of program names (compilers, interpreters, linters etc.) used for executing and analysing the generated code,
+    - `string_to_code` being the function `string_to_code.to_turbo_snake.proc`,
+    - `run_code`: the function, which executes the generated code and returns the _standard output_,
```

Comment:
Codacy has a fix for the issue: [Expected: 2; Actual: 4](https://app.codacy.com/gh/vil02/string_to_code_proj/pullRequest?prid=9915707)
```suggestion
  - `run_code`: the function, which executes the generated code and returns the _standard output_,
```